pandas/tests/window/conftest.py

```diff
@@ -42,21 +42,11 @@ def arithmetic_win_operators(request):
     return request.param
 
 
-@pytest.fixture(params=["right", "left", "both", "neither"])
-def closed(request):
-    return request.param
-
-
 @pytest.fixture(params=[True, False])
 def center(request):
     return request.param
 
 
-@pytest.fixture(params=[None, 1])
-def min_periods(request):
```

Comment:
why are u removing this one?

Reply:
No special reason, was only removing if unused. Adding back these window fixtures.